### PR TITLE
Update administer-cluster out-of-resource.md and namespaces.md

### PR DESCRIPTION
**Pull request by ydFu**
* sync with english version in administer-cluster out-of-resource.md and namespcaes.md

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/namespaces.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ content_type: task
 <!--
 This page shows how to view, work in, and delete {{< glossary_tooltip text="namespaces" term_id="namespace" >}}. The page also shows how to use Kubernetes namespaces to subdivide your cluster.
 -->
-本页展示如何查看、使用和删除{{< glossary_tooltip text="名字空间" term_id="namespace" >}}。
+本页展示如何查看、使用和删除 {{< glossary_tooltip text="名字空间" term_id="namespace" >}} 。
```
Comment (howieyuen) [inline content/zh/docs/tasks/administer-cluster/namespaces.md:17]: 此处显示的是“名字空间”，都是中文，所以这里不需要额外添加空格

File: content/zh/docs/tasks/administer-cluster/namespaces.md [hunk 2]
```diff
@@ -14,19 +14,19 @@ content_type: task
...
 本页同时展示如何使用 Kubernetes 名字空间来划分集群。
 
 ## {{% heading "prerequisites" %}}
 
 <!--
 * Have an [existing Kubernetes cluster](/docs/setup/).
-* Have a basic understanding of Kubernetes _[Pods](/docs/concepts/workloads/pods/pod/)_, _[Services](/docs/concepts/services-networking/service/)_, and _[Deployments](/docs/concepts/workloads/controllers/deployment/)_.
+* You have a basic understanding of Kubernetes {{< glossary_tooltip text="Pods" term_id="pod" >}}, {{< glossary_tooltip term_id="service" text="Services" >}}, and {{< glossary_tooltip text="Deployments" term_id="deployment" >}}.
 -->
 * 你已拥有一个[配置好的 Kubernetes 集群](/zh/docs/setup/).
```
Comment (howieyuen) [inline content/zh/docs/tasks/administer-cluster/namespaces.md:26]: 把这里的英文句号改成中文

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Review by ydFu — COMMENTED** [review]
thank

File: content/zh/docs/tasks/administer-cluster/namespaces.md (hunk 1 above)
Comment (ydFu) [reply content/zh/docs/tasks/administer-cluster/namespaces.md]: thank you for suggesting me.

File: content/zh/docs/tasks/administer-cluster/namespaces.md (hunk 2 above)
Comment (ydFu) [reply content/zh/docs/tasks/administer-cluster/namespaces.md]: thank you for suggesting me.

File: content/zh/docs/tasks/administer-cluster/namespaces.md (hunk 1 above)
Comment (ydFu) [reply content/zh/docs/tasks/administer-cluster/namespaces.md]: thank you for suggesting me.

File: content/zh/docs/tasks/administer-cluster/namespaces.md [hunk 3]
```diff
@@ -21,12 +21,12 @@ This page shows how to view, work in, and delete {{< glossary_tooltip text="name
...
+* You have a basic understanding of Kubernetes {{< glossary_tooltip text="Pods" term_id="pod" >}}, {{< glossary_tooltip term_id="service" text="Services" >}}, and {{< glossary_tooltip text="Deployments" term_id="deployment" >}}.
 -->
-* 你已拥有一个[配置好的 Kubernetes 集群](/zh/docs/setup/).
-* 你已对 Kubernetes 的 _[Pods](/zh/docs/concepts/workloads/pods/)_, 
-  _[Services](/zh/docs/concepts/services-networking/service/)_, 和
-  _[Deployments](/zh/docs/concepts/workloads/controllers/deployment/)_ 有基本理解。
+* 你已拥有一个[配置好的 Kubernetes 集群](/zh/docs/setup/)。
+* 你已对 Kubernetes 的 {{< glossary_tooltip text="Pods" term_id="pod" >}} , 
+  {{< glossary_tooltip term_id="service" text="Services" >}} , 和
+ {{< glossary_tooltip text="Deployments" term_id="deployment" >}} 有基本理解。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/namespaces.md:29]: 29 行应按 28 行缩进两个空格

**Comment by tengqm** [comment]
Umbrella issue for tasks pages is #26178

File: content/zh/docs/tasks/administer-cluster/namespaces.md (hunk 3 above)
Comment (ydFu) [reply content/zh/docs/tasks/administer-cluster/namespaces.md]: thank you for suggesting me.

**Pull request merged**